pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-spark-2.4/pom.xml

```diff
@@ -108,6 +108,12 @@
       <groupId>org.codehaus.woodstox</groupId>
       <artifactId>stax2-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pinot</groupId>
+      <artifactId>pinot-batch-ingestion-spark-base</artifactId>
```

Comment:
You may replace `pinot-batch-ingestion-common` with `pinot-batch-ingestion-spark-base` in the above dependency. No need to specify version or scope explicitly